### DISCUSSION TITLE
Integrating React Components into Django Templates

### DISCUSSION THREAD

**Discussion by burakakca** [kaedroho/django-bridge#46] [Ideas]
I have an existing Django project that uses Django templates, jQuery, and Webpack. I want to add a Django bridge and am trying to find a way to render a React view within certain Django templates. It's not going to be a whole page; it's just going to be a component.

**Comment by kaedroho** [comment]
Thanks for submitting an issue @burakakca!

I've transitioned this to a discussion as adding React components to Django templates isn't currently in scope for Django Bridge. Most of what Django bridge currently provides is the ability to use Django routing in a React project and serialisation of responses from views.

So what I think we should discuss here is whether it makes sense for this project to take on this use-case as well, or if this is something that would be better served as a separate project. Things to consider would be whether any code could be re-used, and how we might clearly document both approaches.
